packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -128,6 +142,20 @@ Terra.describeViewports('Form-Input', ['tiny', 'large'], () => {
 
       Terra.validates.element('input field disabled');
     });
+
+    it('should display InputField supported types', () => {
+      browser.url('raw/tests/terra-form-input/form-input/supported-types-input-field');
+      $('#root').moveTo({ xOffset: 0, yOffset: 700 }); // add to ensure consistent hover styles
+
+      Terra.validates.screenshot('input field supported types');
+    });
+
+    it('should display InputField unsupported types', () => {
+      browser.url('raw/tests/terra-form-input/form-input/unsupported-types-input-field');
+      $('#root').moveTo({ xOffset: 0, yOffset: 700 }); // add to ensure consistent hover styles
+
+      Terra.validates.element('input field unsupported types');
+    });
```

Comment:
Similar as comment above: 

I am wondering about the usefulness of having WDIO snapshots for both of these tests for InputFields, since the test is really meant primarily for unscripted testing and validation. 

I would suggest removing these two tests, since all the types don't all fit within the image capture area (screenshots don't capture all of the types included), and we already are validating the styling of the supported types on the standalone input test.

Reply:
Removed the WDIO tests here https://github.com/cerner/terra-core/pull/3386/commits/2d2817ef44b58f6de8d9b1c0912b5958da012b5e